contrib/debian/manpages/bitcoin-cli.1

```diff
@@ -1,12 +1,12 @@
 .TH BITCOIN-CLI "1" "February 2015" "bitcoin-cli 0.10" 
 .SH NAME
-bitcoin-cli \- a remote procedure call client for Bitcoin Core. 
+bitcoin-cli \- a remote procedure call client for Bitcoin Classic. 
```

Comment:
I suppose the client works for Core as well? Should it be worded it more generally?